macos/FluentUI/Button/Button.swift

```diff
@@ -546,6 +591,18 @@ public enum ButtonStyle: Int, CaseIterable {
 	case borderless
 }
 
+/// Indicates the state of the button for providing different images
+@objc(MSFButtonState)
+public enum ButtonState: Int, CaseIterable {
+	case normal
+	
+	case selected
```

Comment:
how is this different from pressed?

Reply:
Asked @rdeassis, selected is not used but is meant for something like a checkbox. So we should probably name the state pressed here